### PR TITLE
(DOCSP-19563): Add support for Retry Custom Confirmation Functions + Copy improvements to Manage Email Pass Users [Node/RN]

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-19563

### Staged Changes (Requires MongoDB Corp SSO)

- [Manage Email/Password Users - React Native SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-19563-custom-confirm-fix-node-rn/sdk/react-native/examples/manage-email-password-users/)
- [Manage Email/Password Users - Node.js SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-19563-custom-confirm-fix-node-rn/sdk/node/examples/manage-email-password-users/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/node/examples/manage-email-password-users.txt [hunk 1]
```diff
@@ -57,10 +57,55 @@ New users must confirm that they own their email address before they can log in
...
+To confirm a registered user, pass a valid ``token`` and ``tokenId`` to
+:js-sdk:`EmailPasswordAuth.confirmUser()
+<Realm.Auth.EmailPasswordAuth.html#confirmUser>`.
+
+.. tabs-realm-languages::
+
+   .. tab::
+      :tabid: javascript
+
+      .. code-block:: javascript
```
Comment (MongoCaleb) [inline source/sdk/node/examples/manage-email-password-users.txt:86]: nit: should these be bluehawked?

File: source/sdk/node/examples/manage-email-password-users.txt [hunk 2]
```diff
@@ -92,42 +137,32 @@ To send a new confirmation email to a user, pass their email address to
...
    .. tab::
       :tabid: javascript
-
+      
       .. code-block:: javascript
-
-         await app.emailPasswordAuth.confirmUser({ token, tokenId });
-
+         
+         const email = "someone@example.com"; // The user's email address
```
Comment (MongoCaleb) [inline source/sdk/node/examples/manage-email-password-users.txt:156]: same nit as above

**Review by MongoCaleb — APPROVED** [review]
LGTM with question about bluehawking

File: source/sdk/node/examples/manage-email-password-users.txt (hunk 1 above)
Comment (mongomoe) [reply source/sdk/node/examples/manage-email-password-users.txt:86]: yeah they should be, this whole page needs to be unit tested + bluehawked, but for sake of simplicity + time + scope creep I chose to go with the inline code already presented. Bluehawking the rest of the JS docs source is an upcoming project

**Pull request merged**